lib/CMakeLists.txt

```diff
@@ -39,7 +40,7 @@ set_target_properties (libpc-static
     OUTPUT_NAME "pc"
     PREFIX "lib"
     CLEAN_DIRECT_OUTPUT 1
-    COMPILE_FLAGS -fPIC
+    COMPILE_FLAGS "-fPIC -std=gnu89"
```

Comment:
for `qsort_r`
Unfortunately my implementation needs a context pointer to store the sorting dimensions and the standard `qsort` is not reentrant and seems to ask for a global variable to store the context.
Alternatives qsort_r or qsort_s appear to be non-standard. (sort_r)[noporpoise/sort_r], seems to be a relevant polyfill, but I have not tried it yet.